cmd/lakectl/cmd/fs.go

```diff
@@ -159,6 +159,36 @@ var fsUploadCmd = &cobra.Command{
 	},
 }
 
+var fsStageCmd = &cobra.Command{
+	Use:    "stage <path uri>",
+	Short:  "stage an object at the specified URI",
+	Hidden: true,
+	Args: cmdutils.ValidationChain(
+		cobra.ExactArgs(1),
+		cmdutils.FuncValidator(0, uri.ValidatePathURI),
+	),
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getClient()
+		pathURI := uri.Must(uri.Parse(args[0]))
+		size, _ := cmd.Flags().GetInt64("size")
+		location, _ := cmd.Flags().GetString("location")
+		cksum, _ := cmd.Flags().GetString("checksum")
```

Comment:
wish our adapter can fetch the size and checksum from the location at some point in time - but for basic plumbing, this is enough for now.